.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: scala
 jdk: oraclejdk8
 script:
-  - sbt ++2.12.4 memoryBenchmark/compile +test ++2.13.0-M2 timeBenchmark/compile collections-contribJVM/test junit/test scalacheck/test
```

Comment:
`+test` doesn’t do what is supposed to do because `crossScalaVersions` is scoped to a project, not to the build. As a workaround, I explicitly set with which scala version we should compile the project.